doc/developer/feature-benchmark.md

```diff
@@ -60,6 +66,10 @@ Update                    |       0.781 |       0.803 | about the same -2.7%
 
 The table is printed periodically as new rows arrive so that the information is not lost in case of failure.
 
+# Available scenarios
```

Comment:
I agree we should point at the authoritative source so this doesn't rot, but I think it may be worth explaining the top level of the hierarchy: Scenario vs SecnarioBig, the latter doesn't run by default. Also possible that this is sufficiently covered by my nit above on "To run all benchmarks:"